pwned-check/README.md

```diff
@@ -0,0 +1,74 @@
+# Pwned credentials check
+
+Module that provides functionality of checking if the credentials being used for login attempt have been compromised in a data breach.
+
+---
+#### Privacy note:
```

Comment:
I wouldnt call it a privacy note. It is used usually when we have a statement on what we will do with some data that a client gives us.